### DISCUSSION TITLE
Arrows + color coding in PSAT CLI

### DISCUSSION THREAD

**Discussion by helenafranca** [GoogleChromeLabs/ps-analysis-tool#648] [Ideas]
Problem: if I recall correctly, today we don't have visuals (e.g. arrows + color coding) available in the aggregated cookies report of PSAT CLI. And I feel I lose information when I'm looking at the aggregated report (specially related to first-party cookies that are broken).
Request: add this information (either in new columns or using visuals). The aggregated report is the best!

**Comment by milindmore22** [comment]
Hello @helenafranca,

Thanks for letting us know about the lack of visual indicators in the PSAT CLI’s aggregated cookies report. We understand that this can make it more difficult to analyze cookies, particularly for first-party cookies.

We acknowledge this limitation in PSAT CLI and plan to address it with request/response arrows and color coding in a future update.

**Comment by amedina** [comment]
+1 to @milindmore22 points. @helenafranca, these capabilities have been prioritized and will be part of the CLI very soon.
